src/components/AccountSelector.tsx

```diff
@@ -30,7 +30,7 @@ const Main = () => {
       <label style={{ fontSize: '12px' }}>
         <SelectedAccountDiv>{activeAccount}</SelectedAccountDiv>
         <LevelStatusDiv>
-          <label>JOURNEY: {level.toUpperCase()}</label>
+          <StyledLevel>JOURNEY: {level.toUpperCase()}</StyledLevel>
```

Comment:
I recommend using the bootstrap semantic tags, as it will help us make spacing consistent in the app:

```suggestion
          <p className="pr-4">JOURNEY: {level.toUpperCase()}</p>
```